src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ExtensionMethodImportCompletionHelper.SymbolComputer.cs

```diff
@@ -86,7 +86,7 @@ public static async ValueTask UpdateCacheAsync(Project project, CancellationToke
                     await GetUpToDateCacheEntryAsync(relevantProject, cacheService, cancellationToken).ConfigureAwait(false);
 
                 foreach (var peReference in GetAllRelevantPeReferences(project))
-                    await SymbolTreeInfo.GetInfoForMetadataReferenceAsync(project.Solution, peReference, loadOnly: false, cancellationToken).ConfigureAwait(false);
```

Comment:
the loadOnly flag entirely went away.  There are now just two distinct types of callers.  Those that want the completely up-to-date index (and are willing to compute it), and those that are are ok with a stale index and will either use what is already there, or will try to load whatever index we last stored to disk if not.